realm/realm-library/src/main/java/io/realm/RealmQuery.java

```diff
@@ -1251,25 +1251,29 @@ public boolean isValid() {
     // Like
 
     /**
-     * Condition that the value of field matches with the specified substring, with wildcards.
+     * Condition that the value of field matches with the specified substring, with wildcards:
+     * '*' matches [0, n] unicode chars
```

Comment:
Wrap in HTML list `ul/li`. Otherwise the Javadoc generator will just put this on one line.